requirements.txt

```diff
@@ -1,21 +1,35 @@
+aiohttp==3.9.5
```

Comment:
I would recommend swapping totally into 1 HTTP library rather than split across 3 (aiohttp, requests, httpx) - Or, start moving from requests -> (aiohttp or httpx)

Reply:
Thank you so much Bryan for all your help on this PR! I have updated the code to use httpx instead of aiohttp. Thanks again.